org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/typesystem/IResolvedTypes.java

```diff
@@ -70,7 +70,7 @@
 	 * Returns the actually inferred return type for the given expression or <code>null</code> if none.
 	 * Returns the actual expression type, if no explicit return type exists.
 	 * 
-	 * Shorthand for {{@link #getReturnType(XExpression, false)}}
+	 * Shorthand for <code>getReturnType(XExpression, false)</code>
```

Comment:
As discussed. `{@link #getReturnType(XExpression, false)}` would be my preference.

Reply:
Which should have read 
`{@link #getReturnType(XExpression, boolean) getReturnType(expression, false)}`